### DISCUSSION TITLE
Welcome to layui-vue Discussions!

### DISCUSSION THREAD

**Discussion by sleeprite** [lunar-landing/layui-vue#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by miko-chen** [comment]
- i want sleep 

<div>
<img src="https://tvax4.sinaimg.cn/large/007YLcQ6ly1h44l660p0xj31hc0u0thy" alt="巨人堤道上的玄武岩柱巨人堤道" class="image top-main-bg" id="mbg" title="巨石中的巨石 点击全屏预览">
</div>

**Comment by Rain-yry** [comment]
下拉选择的搜索框清除功能会导致数据丢失

**Reply by sleeprite** [reply]
很抱歉现在才回复，大概过去了半年左右，layui-vue 已经更新了几百个特性和问题，你可以更新到最新版本查看问题是否还存在。

目前以 Gitee 处理 issues 为主，Github 难免会忽略。
